src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx

```diff
@@ -144,7 +145,16 @@ function BaseReportActionContextMenu({
 
     const childReport = ReportUtils.getReport(reportAction?.childReportID ?? '-1');
     const parentReportAction = ReportActionsUtils.getReportAction(childReport?.parentReportID ?? '', childReport?.parentReportActionID ?? '');
-    const {reportActions: paginatedReportActions} = usePaginatedReportActions(childReport?.reportID ?? '-1');
+    const {reportActions: paginatedReportActions, sortedAllReportActions} = usePaginatedReportActions(childReport?.reportID ?? '-1');
+    const hasNewestReportAction = sortedAllReportActions && sortedAllReportActions[0]?.created === childReport?.lastVisibleActionCreated;
+
+    useEffect(() => {
+        if (hasNewestReportAction) {
+            return;
+        }
+
+        Report.openReport(childReport?.reportID ?? '-1');
```

Comment:
```suggestion
        if (hasNewestReportAction || !childReport?.reportID) {
            return;
        }

        Report.openReport(childReport.reportID);
```

Reply:
Sure, I've replaced it with an early return instead of using the nullish coalescing operator.